torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1335,19 +1335,64 @@ const std::vector<std::string> functions = {
         def clamp(self,
                   min: Optional[number],
                   max: Optional[number]):
-          def backward(grad_output):
-            if min is not None and max is not None:
-                mask = ((self >= float(min)) * (self <= float(max))).type_as(self)
-                return grad_output * mask, None, None
-            elif min is not None:
-                mask = (self >= float(min)).type_as(self)
-                return grad_output * mask, None, None
-            elif max is not None:
-                mask = (self <= float(max)).type_as(self)
-                return grad_output * mask, None, None
-            else: #min is None and max is None
-                return grad_output, None, None
-          return torch.clamp(self, min=min, max=max), backward
+            def backward(grad_output):
```

Comment:
Is this overindented?

Reply:
I don't think it is, but I will check it on an IDE. I think it was previously underindented and indented it here to fix it.

Reply:
Cool. Not a big issue either way.